index.bs

```diff
@@ -703,7 +703,7 @@ The <code>'[=ster=]'</code> entity group as defined in [[!HEIF]] may be used to
 
 <h3 id="brands-overview">Brands overview</h3>
 
-<p>As defined by [[!ISOBMFF]], the presence of a brand in the <code>[=compatible_brands=]</code> list in the <code>[=FileTypeBox=]</code> can be interpreted as the permission for those [=AV1 Image File Format=] readers/parsers and [=AV1 Image File Format=] renderers that only implement the features required by the brand, to process the corresponding file and only the parts (e.g. items or sequences) that comply with the brand.</p>
+<p>As defined by [[!ISOBMFF]], the presence of a brand in the <code>[=major_brand=]</code> field or in the <code>[=compatible_brands=]</code> list in the <code>[=FileTypeBox=]</code> can be interpreted as the permission for those [=AV1 Image File Format=] readers/parsers and [=AV1 Image File Format=] renderers that only implement the features required by the brand, to process the corresponding file and only the parts (e.g. items or sequences) that comply with the brand.</p>
```

Comment:
Do we want to simplify these changes and simply say something along the lines:
"As specified in ISOBMFF, the major brand shall be treated as if it is present among the compatible brands even if not explicitly specified."

Thoughts?

Reply:
It would simplify this PR but not the spec itself in my opinion. It would make it slightly shorter though.

Maybe just say something such as "if `avif` is present in `ftyp`" or similar? This way it can implicitly be in `major_brand` and/or `compatible_brands` (but there could be confusion about `minor_version`).

Reply:
Note that a similar wording already exists:

> Conversely, if such a track exists and the constraints of the brand [avio](https://github.com/AOMediaCodec/av1-avif/pull/264#avif-intra-only-brand-avio) on [AV1 image items](https://github.com/AOMediaCodec/av1-avif/pull/264#av1-image-item) are met, the brand should be used.

Reply:
> Maybe just say something such as "if avif is present in ftyp" or similar? This way it can implicitly be in major_brand and/or compatible_brands (but there could be confusion about minor_version).

I like that. I don't _think_ anyone will confuse it with `minor_version`, but I guess we can add a sentence at the top to say that `minor_version` does not count as a brand or something like that.

Reply:
From meeting: new PR that tries to make it more general and just mentions having the brand in the `ftyp`.

Reply:
Done in https://github.com/AOMediaCodec/av1-avif/pull/289.